rfcs/formats/identity.md

```diff
@@ -0,0 +1,203 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# Identity Plane Data Formats
+
+* Authors: Bishakh Chandra Ghosh, Venkatraman Ramakrishna, Krishnasuri Narayanam
+* Status: Draft
+* Since: 25-July-2021
+
+
+## Participant Unit Identity
+
+Each network participant (individual/organization) has its identity on the identity plane as *Participant Unit Identity*, in the form of a **DID**. We also refer to it as *Participant Unit DID*.
+The participant DID Document must specify one verification method with [authentication](https://w3c.github.io/did-core/#authentication) verification relationship. The *Participant Unit DID* must be registered in some IIN registry.
+
+### Example:
+
+```json
+{
+  "id": "did:exampleiin:org1",
+  "verificationMethod": [{
+    "id": "did:exampleiin:org1#key1",
+    "type": "Ed25519VerificationKey2020",
+    "controller": "did:exampleiin:org1",
+    "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+  }],
+
+  "authentication": ["did:exampleiin:org1#key1"]
+}
+```
+
+
+## Network Identity
+
+The identity of a network as a single entity is represented by the *Network Identity* which is a DID called *Network DID*.
+
+A permissioned blockchain network is formed by mutual agreement among its participants. The network as an entity is thus not controlled by any single member but together by its participant units, which may be individuals or companies or organizations. But inspite of having multiple controllers, the network as an entity must have its own identity, through which it can be discovered. In this specification, each permissioned blockchain network which needs to exchange its identity for interoperation has a Network DID and a corresponding Network DID Document, that we describe in this section.
```

Comment:
Following "among its participants.", we can add examples: "Participants, whether they be in a Fabric, Corda, or Ethereum network, typically control one or more peers and act as identity providers for those peers and associated clients. In Fabric, the notion of a participant is formalized in the role of an _organization_."